site/model_help.md

```diff
@@ -0,0 +1,99 @@
+## The Model Help Tool
+
+The Model Help Tool provides information about the folders and attributes that are valid for sections and folders of a domain model. This is useful when creating a new domain model, or expanding an existing model, including discovered models.
+
+**NOTE:** The Model Help Tool was added as part of WebLogic Deploy Tooling 1.8.
```

Comment:
was added as part of WebLogic Deploy Tooling 1.8. -> was added in WebLogic Deploy Tooling 1.8. OR, you can state, "The Model Help Tool is new in WebLogic Deploy Tooling 1.8."